src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/order_actions.html.twig

```diff
@@ -61,6 +61,15 @@
     </form>
   {% endif %}
 
+  <form class="form-inline d-inline-block form-horizontal ml-2">
+    <div class="input-group">
+      <button type="button" class="btn btn-action js-print-order-view-page">
+        <i class="material-icons">print</i>
```

Comment:
Missing aria attribute (this should be done for ALL decorative icons)
```suggestion
        <i class="material-icons" aria-hidden="true">print</i>
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17617/commits/880b0ffdaca18f032d2cb941f37e0751b4745479